src/contracts/facilitators/aave/misc/GhoManager.sol

```diff
@@ -0,0 +1,73 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.10;
+
+import {IACLManager} from '@aave/core-v3/contracts/interfaces/IACLManager.sol';
+import {IPoolAddressesProvider} from '@aave/core-v3/contracts/interfaces/IPoolAddressesProvider.sol';
+import {Errors} from '@aave/core-v3/contracts/protocol/libraries/helpers/Errors.sol';
+import {PoolConfigurator} from '@aave/core-v3/contracts/protocol/pool/PoolConfigurator.sol';
+
+// Gho Contracts
+import {GhoVariableDebtToken} from 'src/contracts/facilitators/aave/tokens/GhoVariableDebtToken.sol';
+
+/**
+ * @title GhoManager
+ * @author Aave
+ * @notice GhoManager contract for setting Gho interest rate strategy, discount rate strategy, and discount lock period
+ */
+contract GhoManager {
+  // The Access Control List manager contract
+  IACLManager private immutable ACL_MANAGER;
+
+  /**
+   * @dev Only risk or pool admin can call functions marked by this modifier.
+   */
+  modifier onlyGhoManager() {
```

Comment:
This modifier is not needed. Risk/Pool admins are already able to call these functions.
We need to make this contract Ownable, so only the owner can call is functions

Reply:
removed